config/defaults.mk

```diff
@@ -365,7 +365,13 @@ RAJA_OPT = -I$(RAJA_DIR)/include
 ifdef CUB_DIR
    RAJA_OPT += -I$(CUB_DIR)
 endif
+ifeq ($(MFEM_USE_HIP),YES)
+RAJA_LIB = -L$(RAJA_DIR)/lib -lRAJA -ldl
```

Comment:
@camierjs for some reason I had to make this modification to include RAJA when HIP is enabled. Could it be a clang-hip thing?

Reply:
@artv3 I'm running into this same issue with some of my HIP porting efforts of ExaConstit with clang-hip.

Reply:
@nbeams made some changes in her branch that makes my modifications unnecessary.